### ISSUE TITLE
SVG does not render after recent parsing rewrite

### ISSUE THREAD

**Issue by jbg** [dnfield/flutter_svg#100]
Since the recent rewrite of parsing, the following SVG (DE country flag) does not render (nothing at all is rendered). I plan to look into this further and will update the issue with more info.

```svg
<svg xmlns="http://www.w3.org/2000/svg"
     width="5"
     height="3"
     viewBox="0 0 5 3"
     preserveAspectRatio="xMidYMid"
     xmlns:usvg="https://github.com/RazrFalcon/usvg"
     usvg:version="0.4.0">
    <defs/>
    <path id="black_stripe"
          fill="#000000"
          fill-opacity="1"
          fill-rule="nonzero"
          stroke="none"
          visibility="visible"
          d="M 0 0 L 5 0 L 5 3 L 0 3 Z" />
    <path id="red_stripe"
          fill="#dd0000"
          fill-opacity="1"
          fill-rule="nonzero"
          stroke="none"
          visibility="visible"
          d="M 0 1 L 5 1 L 5 3 L 0 3 Z" />
    <path id="gold_stripe"
          fill="#ffce00"
          fill-opacity="1"
          fill-rule="nonzero"
          stroke="none"
          visibility="visible"
          d="M 0 2 L 5 2 L 5 3 L 0 3 Z" />
</svg>
```

**Comment by dnfield** [comment]
It's the empty `<defs/>` element.  Will fix...

**Comment by dnfield** [comment]
Fixed in 0.10.0+1 (c75f423)

**Comment by jbg** [comment]
Thanks!
